tests/integration.rs

```diff
@@ -3912,4 +3912,45 @@ mod tests {
             Ok(())
         });
     }
+
+    #[test]
+    fn test_pvh_initramfs() {
+        test_block!(tb, "", {
+            let mut clear = ClearDiskConfig::new();
+            let guest = Guest::new(&mut clear);
+            let mut workload_path = dirs::home_dir().unwrap();
+            workload_path.push("workloads");
+
+            let mut kernel_path = workload_path.clone();
+            kernel_path.push("vmlinux.pvh");
+
+            let mut initramfs_path = workload_path;
+            initramfs_path.push("alpine_initramfs.img");
+
+            let test_string = String::from("axz34i9rylotd8n50wbv6kcj7f2qushme1pg");
+            let cmdline = format!("console=hvc0 quiet TEST_STRING={}", test_string);
+
+            let mut child = GuestCommand::new(&guest)
+                .args(&["--kernel", kernel_path.to_str().unwrap()])
+                .args(&["--initramfs", initramfs_path.to_str().unwrap()])
+                .args(&["--cmdline", &cmdline])
+                .capture_output()
+                .spawn()
+                .unwrap();
+
+            thread::sleep(std::time::Duration::new(20, 0));
+
+            let _ = child.kill();
+            match child.wait_with_output() {
+                Ok(out) => {
+                    let s = String::from_utf8_lossy(&out.stdout);
+                    println!("{}", s);
+                    aver_ne!(tb, s.lines().position(|line| line == test_string), None);
+                }
+                Err(_) => aver!(tb, false),
+            }
+
+            Ok(())
+        });
+    }
```

Comment:
Sorry to be picky but could you factorize this into the initial test `test_initramfs`? I think the only difference is the vmlinux name, so that'd be nice to have a simple loop running the same test for a list of kernels.
A bit similar to the way `test_simple_launch` runs over multiple images.

Reply:
Makes sense, working on it..

Reply:
Followed same approach as `test_simple_launch` (thanks for the advice). Let me know if I missed anything.

Reply:
Great thanks!